### PR TITLE
UIORGS-356 - Show in version history record view, which fields have been edited

### PR DESCRIPTION
**Pull request by usavkov-epam**
## Purpose

https://folio-org.atlassian.net/browse/UIORGS-356

 Give the user the ability to access the change log that displays all edits that have been made to the record.

## Approach

Apply ACQ approach to display version history (`ui-orders` app is a prototype).

Prepare organization components views adapted for version history context. Implement a hook, that will aggregate version history data for specific versions.

## Screenshot

https://github.com/user-attachments/assets/e43e0cc5-4d4a-46a8-a6b0-14ab6e77f8f6

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/Organizations/OrganizationVersion/components/OrganizationContactInfoVersionView/components/ContactCardHeaderVersionView/ContactCardHeaderVersionView.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import css from './ContactCardHeaderVersionView.css';
+
+export const ContactCardHeaderVersionView = ({ isPrimary, name }) => {
+  const versionContext = useContext(VersionViewContext);
+
+  const isUpdated = versionContext?.paths?.includes(name);
+
+  const headerStart = (
+    <h4 className={css.cardHeader}>
+      <span className={isUpdated ? css.mark : ''}><FormattedMessage id={`ui-organizations.${isPrimary ? 'primaryItem' : 'alternateItem'}`} /></span>
```
Comment (alisher-epam) [inline src/Organizations/OrganizationVersion/components/OrganizationContactInfoVersionView/components/ContactCardHeaderVersionView/ContactCardHeaderVersionView.js:16]: can be set into a new line
```suggestion
      <span className={isUpdated ? css.mark : ''}>
         <FormattedMessage id={`ui-organizations.${isPrimary ? 'primaryItem' : 'alternateItem'}`} />
       </span>
```

File: src/Organizations/OrganizationVersion/components/OrganizationContactInfoVersionView/components/ContactPersonPhonesVersionView.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+} from '@folio/stripes-acq-components';
+
+import { ContactPersonSection } from '../../../../../ContactPeople/ContactPerson';
+import { ContactCardHeaderVersionView } from './ContactCardHeaderVersionView';
+
+export const ContactPersonPhonesVersionView = ({ phones }) => {
+  if (!phones?.length) return null;
+
+  const renderBody = () => (
+    (
```
Comment (alisher-epam) [inline src/Organizations/OrganizationVersion/components/OrganizationContactInfoVersionView/components/ContactPersonPhonesVersionView.js:21]: redundant

File: src/Organizations/OrganizationVersion/components/OrganizationContactPeopleVersionView/OrganizationContactPeopleVersionView.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
+import get from 'lodash/get';
+import find from 'lodash/find';
```
Comment (alisher-epam) [inline src/Organizations/OrganizationVersion/components/OrganizationContactPeopleVersionView/OrganizationContactPeopleVersionView.js:2]: swap the order, please

File: src/common/hooks/useInterfacesByIds/useInterfacesByIds.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+const DEFAULT_DATA = [];
+
+export const useInterfacesByIds = (interfaceIds, options = {}) => {
+  const {
+    enabled = true,
+    ...queryOptions
+  } = options;
+
+  const ky = useOkapiKy();
+  const [namespace] = useNamespace('interfaces');
```
Comment (alisher-epam) [inline src/common/hooks/useInterfacesByIds/useInterfacesByIds.js:20]:
```suggestion
  const [namespace] = useNamespace({ key: 'interfaces' });
```

**Review by alisher-epam — APPROVED** [review]

**Review by SerhiiNosko — APPROVED** [review]
Great and huge work done here

**Pull request merged**